src/block/put.js

```diff
@@ -55,7 +55,17 @@ module.exports = (send) => {
 
     sendOneFile(block.data, { qs }, (err, result) => {
       if (err) {
-        return callback(err) // early
+        // Retry with "protobuf" format for go-ipfs
+        // TODO: remove when https://github.com/ipfs/go-cid/issues/75 resolved
+        if (qs.format === 'dag-pb') {
+          qs.format = 'protobuf'
```

Comment:
This is the key change for me that would have allowed me to complete #815.  Still learned a lot from trying to work that story.  Time to find a new issue to dig into. :)